lib/perl/Genome/Model/Build/ClinSeq/InputBuilds.pm

```diff
@@ -30,4 +30,9 @@ sub resolve_rnaseq_builds {
   $rnaseq_builds->{$normal_build->id}{type} = 'rnaseq' if $normal_build;
 }
 
+sub wgs_exome_build {
```

Comment:
I agree with @jasonwalker80 that this is out of scope for this PR. I do think we should make the subroutine names consistent once we do the bigger ClinSeq refactor.